xskillscore/core/comparative.py

```diff
@@ -0,0 +1,151 @@
+import warnings
+
+import scipy.stats as st
+import xarray as xr
+
+
+def sign_test(
+    observation,
+    forecast1,
+    forecast2,
+    dim=None,
+    alpha=0.05,
+    metric=None,
+    orientation='negative',
+):
+    """
+        Returns the Delsole and Tippett sign test over the given time dimension.
+
+        Parameters
+        ----------
+        observation : xarray.Dataset or xarray.DataArray or None
+            observation to be compared to both forecasts.
+            If ``None``, then assume that forecast1 and forecast2 have already been
+            compared to observation. If metric is None, this assumes forecasts to be
+            already compared to observation before and ignores observation. Please
```

Comment:
Looks like this statement is repeated twice here.

Reply:
its not